src/3d/qgs3dmapscene.cpp

```diff
@@ -324,9 +324,9 @@ void Qgs3DMapScene::onCameraChanged()
   if ( mMap.projectionType() == Qt3DRender::QCameraLens::OrthographicProjection )
   {
     QRect viewportRect( QPoint( 0, 0 ), mEngine->size() );
-    const float viewWidthFromCenter = mEngine->camera()->position().distanceToPoint( mEngine->camera()->viewCenter() );
+    const float viewWidthFromCenter = mCameraController->distance();
     const float viewHeightFromCenter =  viewportRect.height() * viewWidthFromCenter / viewportRect.width();
-    mEngine->camera()->lens()->setOrthographicProjection( -viewWidthFromCenter, viewWidthFromCenter, -viewHeightFromCenter, viewHeightFromCenter, 1.0f, 10000.0f );
+    mEngine->camera()->lens()->setOrthographicProjection( -viewWidthFromCenter, viewWidthFromCenter, -viewHeightFromCenter, viewHeightFromCenter, 10.0f, 10000.0f );
```

Comment:
@nirvn Sorry to bug you regarding near/far planes again - I think we should keep previously used values from mEngine->camera() rather than setting the range to 10 - 10000

Reply:
@wonder-sk , sorry, I misunderstood you, I thought you only meant for me to revert to 10.0f as a near plane value. 

I used a hard-coded value to mimic what's on line 102 in the same source file: https://github.com/nirvn/QGIS/blob/1fe05fedcecbbf0fc56620b7621058e53b3caa48/src/3d/qgs3dmapscene.cpp#L102 -- do you want me to change that line too?

Reply:
Those values 10 to 10000 in the constructor are a bit arbitrary (and we can keep them). After we create 3D scene we set the view from the top and initialize near/far planes to more appropriate values (see QgsCameraController::setViewFromTop()), which are later continuously updated as camera moves (see Qgs3DMapScene::updateCameraNearFarPlanes()). So best to just change the values in setOrthographicProjection() to use previously used near/far planes